e2e/atlas/dbusers_test.go

```diff
@@ -55,27 +70,30 @@ func TestDBUsers(t *testing.T) {
 			"atlasAdmin",
 			"--deleteAfter", time.Now().AddDate(0, 0, 1).Format(time.RFC3339),
 			"--username", username,
-			"--password=passW0rd",
+			"--password", userPassword,
 			"--scope", scopeClusterDataLake,
 			"-o=json",
 		)
-		cmd.Env = os.Environ()
-		resp, err := cmd.CombinedOutput()
-		a := assert.New(t)
-		a.NoError(err)
 
-		var user mongodbatlas.DatabaseUser
-		if err := json.Unmarshal(resp, &user); err != nil {
-			t.Fatalf("unexpected error: %v", err)
-		}
+		testCreatePasswordCmd(t, cmd, username)
+	})
 
-		a.Equal(username, user.Username)
-		if a.Len(user.Scopes, 2) {
-			a.Equal(user.Scopes[0].Name, clusterName0)
-			a.Equal(user.Scopes[0].Type, clusterType)
-			a.Equal(user.Scopes[1].Name, clusterName1)
-			a.Equal(user.Scopes[0].Type, clusterType)
-		}
+	t.Run("CreateWithPasswordFromStdin", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			dbusersEntity,
+			"create",
+			"atlasAdmin",
+			"--deleteAfter", time.Now().AddDate(0, 0, 1).Format(time.RFC3339),
+			"--username", stdinUsername,
+			"--scope", scopeClusterDataLake,
+			"-o=json",
+		)
+
+		passwordStdin := bytes.NewBuffer([]byte(fmt.Sprintf("%s", userPassword)))
```

Comment:
[q] wouldn't this be the same as 
```suggestion
		passwordStdin := bytes.NewBuffer([]byte(userPassword))
```

Reply:
yep, the linting got it